Dockerfile

```diff
@@ -40,7 +40,8 @@ RUN set -ex \
 RUN set -ex \
  && mkdir -p /var/lib/mopidy/.config \
  && ln -s /config /var/lib/mopidy/.config/mopidy
-
+RUN python3 -m pip install Mopidy-Local
```

Comment:
I wonder if you could use Pipfile instead (https://github.com/wernight/docker-mopidy/blob/master/Pipfile) so pipenv would pick it up?